### ISSUE TITLE
Move old blueprints to this repository

### ISSUE THREAD

**Issue by ifraixedes** [storj/design-docs#4]
## Motivation/Why?
- We want to have all the design documents in one place to make it easy for everybody (employees, community members, contributors, etc.) to find them. 

## What needs to be done?
- Move the blueprint documents stored in the  [storj repository](https://github.com/storj/storj/tree/main/docs/blueprints) to this one.

## Acceptance Criteria
- Create a PR with all the documents that are in the [storj repository](https://github.com/storj/storj/tree/main/docs/blueprints).
- Create a commit to delete all the documents of the [storj repository](https://github.com/storj/storj/tree/main/docs/blueprints).

Blocked by #3

**Comment by profclems** [comment]
Closing this as duplicate of #8
